lib/redis_memo/redis.rb

```diff
@@ -43,9 +44,9 @@ def initialize(orig_options)
       super(primary_option)
     end
 
-    alias_method :get_primary, :get
-    alias_method :mget_primary, :mget
-    alias_method :mapped_mget_primary, :mapped_mget
+    alias get_primary get
+    alias mget_primary mget
+    alias mapped_mget_primary mapped_mget
```

Comment:
Is this a rubocop suggestion? I did a quick google search, and it feels like alias_method is a bit more straightforward to use/more predictable

Reply:
Yes it is. I agree with you tho! Let me undo this change and adjust rubocop